docs/platforms/javascript/common/install/esm.mdx

```diff
@@ -10,20 +10,19 @@ supported:
   - javascript.koa
 ---
 
-When running your application in ESM mode, you can't use `require()` to load modules. Instead, you have to use the  `--import` command line options to load a module before the application starts:
-
-Adjust the Node.js call for your application to use the [--import](https://nodejs.org/api/cli.html#--importmodule) parameter and point it at `instrument.js`, which contains your `Sentry.init()` code:
+When running your application in ESM mode, you need to pass an [--import](https://nodejs.org/api/cli.html#--importmodule) command line option to register the SDK's ESM module loader so that it can properly instrument libraries.
```

Comment:
```suggestion
Most node applications today are either written in CommonJS (CJS), or compiled to CJS before being ran. While CommonJS uses `require()` to load modules, we recommend using the `instrument.js` file at the top of your application installation method instead. 

However, if your application runs in ESM mode, this method won't work unless you run your application with `--import @sentry/node/import`. Note, that even if your application is written in ESM (using `import`), it may still be _run_ in CJS. (Almost all applications written in TypeScript are compiled to CJS before being ran, for example.) If this is the case, follow the [CommonJS instructions](../commonjs).

## How to Run Your Application

To run your application in ESM mode, you'll need to pass an [--import](https://nodejs.org/api/cli.html#--importmodule) command line option like the one below to register the SDK's ESM module loader so that it can properly instrument libraries.
```